Lib/asyncio/base_events.py

```diff
@@ -155,6 +156,63 @@ def _run_until_complete_cb(fut):
     futures._get_loop(fut).stop()
 
 
+
+class _SendfileProtocol(protocols.Protocol):
+    def __init__(self, transp):
+        # transport should be _FlowControlMixin instance
+        self._transport = transp
+        self._proto = transp.get_protocol()
+        self._should_resume_reading = transp.is_reading()
+        self._should_resume_writing = transp._protocol_paused
+        transp.pause_reading()
+        transp.set_protocol(self)
+        if self._should_resume_writing:
+            self._paused = self._transport._loop.create_future()
+        else:
+            self._paused = None
+
+    def connection_made(self, transport):
+        raise RuntimeError("Invalid state, "
```

Comment:
This reads better: "Invalid state, " -> "Invalid state: "

ditto for other similar errors